docs/buckets.rst

```diff
@@ -0,0 +1,62 @@
+Buckets
+#######
+
+Buckets are a way to group users around some data.
+
+Basically has got an and a list of principals that can administrate
+it.
```

Comment:
I believe we should have the same set of permissions we have for other objects (read, create, update, delete)